packages-exp/auth-types-exp/index.d.ts

```diff
@@ -214,3 +214,9 @@ export interface UserCredential {
   credential: AuthCredential | null;
   operationType: OperationType;
 }
+
+export interface ApplicationVerifier {
```

Comment:
do you need to add recaptchaverifier here too?

Reply:
No I don't think so; the abstracted type that we care about for putting in here is `ApplicationVerifier`